elliottlib/cli/verify_attached_bugs_cli.py

```diff
@@ -95,12 +108,14 @@ def _get_blocking_bugs_for(self, bugs):
         v = minor_version_tuple(self.target_releases[0])
         next_version = (v[0], v[1] + 1)
 
+        pattern = re.compile(r'^[0-9]+\.[0-9]+\.(0|z)$')
+
         # retrieve blockers and filter to those with correct product and target version
         blocking_bugs = {
             bug.id: bug
             for bug in bzutil.get_bugs(self.bzapi, list(candidate_blockers)).values()
             # b.target release is a list of size 0 or 1
-            if any(minor_version_tuple(target) == next_version for target in bug.target_release)
+            if any(minor_version_tuple(target) == next_version for target in bug.target_release if pattern.match(target))
```

Comment:
> Since we can now pass any bug to be validated, there is a chance that the field
"target_release" is filled with strings in formats that our code is not expecting,
such as "FDP 21.I", found in BZ 1978605.

Reply:
Ah, thanks! Would it make sense to compare `Product` in that case, maybe as addition? Feel less fragile and random than a regex on a version.

Reply:
hm, good point! yeah, makes much more sense ... going to include that check now.

Reply:
done ✔️